libqtile/manager.py

```diff
@@ -630,7 +630,13 @@ def manage(self, w):
                 self.windowMap[w.wid] = c
                 # Window may have been bound to a group in the hook.
                 if not c.group:
-                    self.currentScreen.group.add(c, focus=c.can_steal_focus())
+                    # NOTE: checking self.config.focus_on_mapped_windows will
+                    # fail tests since most of the test configs don't have this
+                    # set and also aren't real configs. Should the test configs
+                    # be read with readconfig like any other config?
+                    steal_focus = getattr(self.config,
```

Comment:
OK, I reversed the getattr and instead added the `focus_on_mapped_windows = True` to 30 config files in `test/`. I feel like I have to be missing something though. It seems to me the config files aught to automatically notice the new option from the confreader or default_config.py